torch/utils/tensorboard/x2num.py

```diff
@@ -0,0 +1,34 @@
+# DO NOT alter/distruct/free input object !
```

Comment:
Is this module intended to be part of the public facing API, or just an internal helper module? If it's an internal helper, let's please prefix it with an underscore so that external users don't start depending on it.

Reply:
renamed to `_convert_np`